README.md

```diff
@@ -180,8 +180,8 @@ Thanks for you want contributing to Jule!
 Every contribution, big or small, to Jule is greatly appreciated.
 <br><br>
 The Jule project use issues for only bug reports and proposals. <br>
-To contribute, please read the contribution guidelines from <a href="https://julelang.github.io/website/pages/contributing.html">here</a>. <br>
-To discussions and questions, please use <a href="https://github.com/julelang/jule/discussions">discussions</a>.
+To contribute, please read the [contribution guidelines](https://julelang.github.io/website/pages/contributing.htm). <br>
```

Comment:
Please change `.htm` to `.html` for correct url.

Reply:
Ahh, yes. My bad :/

Fixed: cebb6ca